src/index.ts

```diff
@@ -8,6 +17,6 @@ export { paletteOptimization } from './optimizers';
 // get palette
 export { getPaletteFromImage, getPaletteFromString } from './extractors';
 // Professional test
```

Comment:
```suggestion
// evaluators
```

???

Reply:
The comment here is to identify the corresponding function, not the path. 

Need to modify it?